### DISCUSSION TITLE
CapacityBrowser iOS built failure with "Type 'UIColor' has no member 'capacitor'"

### DISCUSSION THREAD

**Discussion by mkliu** [ionic-team/capacitor#4231] [Q&A]
I have a brand new app, using capacitor 3 (otherwise get pod spec failure) and enabled iOS.
But it's giving me this error "The UIColor has no member 'capacitor'. The error points to this line:
```
            color = UIColor.capacitor.color(fromHex: toolbarColor)
```

Any suggestions?

**Comment by francescoatseedble** [comment]
Have you found a solution? I'm having the same problem.

**Comment by jcesarmobile** [comment]
make sure you have all capacitor packages (ios/android/cli/core) on 3.0.0 or newer
